source/Halibut.Tests/ParallelRequestsFixture.cs

```diff
@@ -252,17 +252,16 @@ void WaitForAllThreads(List<Thread> threads)
         
         static int NumberOfParallelRequests(ClientAndServiceTestCase clientAndServiceTestCase)
         {
-            int threadCount = 64;
-            if (!clientAndServiceTestCase.ClientAndServiceTestVersion.IsPreviousClient())
+            if (clientAndServiceTestCase.ClientAndServiceTestVersion.IsPreviousClient())
```

Comment:
Lol, didn't I just review this change in the other PR? (with the exception of using 32 instead of 64)

Reply:
Yes, this wouldn't go gree without the associated change :( 

I tried to split changes across 2 PRs but it was 100% successful!!